src/components/ImageUploader.tsx

```diff
@@ -0,0 +1,83 @@
+import { Component, toNative, Vue, Watch, Prop } from 'vue-facing-decorator';
+import { observer } from 'mobx-vue-helper';
+
+import Overlay from './Overlay';
+import styles from './ImageUploader.module.less';
+
+@Component
+@observer
+class ImageUploader extends Vue {
+  @Prop({ type: String }) readonly name?: string;
+  @Prop({ type: Boolean }) readonly required?: boolean;
```

Comment:
```suggestion
  @Prop({ type: String })
  readonly name?: string;

  @Prop({ type: Boolean })
  readonly required?: boolean;
```
装饰器在属性上方，并且留有空行，会阅读清晰。